tests/analyze_assoc.R

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/Rscript
+#
+# Copyright 2015 Google Inc. All rights reserved.
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Reads map files, report files, and RAPPOR parameters to run 
+# an EM algorithm to estimate joint distribution over two or more variables
+# 
+# Usage:
+#       $ Rscript analyze_assoc.R -map1 map_1.csv -map2 map_2.csv \
+#                                 -reports reports.csv \
+# Inputs: map1, map2, reports, params
+#         see how options are parsed below for more information
+# Outputs:
+#         prints a table with estimated joint probability masses
+#         over candidate strings
+#         Ex. 
+#                 ssl   nossl
+#         intel   0.1   0.3
+#         google  0.5   0.1
+
+library("optparse")
+
+# First parse args
+options(stringsAsFactors = FALSE)
+if(!interactive()) {
+  option_list <- list(
+    # Flags
+    make_option(c("--map1", "-m1"), default = "map_1.csv",
+                help = "1st map filename"),
```

Comment:
This help isn't helpful... Something like "hashed candidates for the first variable" would be better.